doc/modules/mod_commands.md

```diff
@@ -0,0 +1,72 @@
+# MongooseIM's command set
+
+## Purpose
+
+This is a basic set of administration and client commands, which provide sort of an API for MongooseIM.
```

Comment:
Our goal is to create a consistent, easy to use API for MongooseIM. Both backend and client commands provide enough information to allow auto-generating the access methods. We currently use it in our admin and client REST API interface. In the future it might replace the current mongooseimclt implementation.